tests/acceptance/test_build.py

```diff
@@ -496,6 +501,12 @@ def test_preferred_versions(self, prepared_test_build, recipe, version):
                         is not None
                     ):
                         continue
+
+                    elif line.startswith("SSTATE_DIR"):
+                        new_fd.write(
+                            "# QA-784: Cache write disabled for this test\n# %s" % line
+                        )
```

Comment:
Won't this, combined with the changes from mender-image-tests, build everything from scratch? If the sstate cache is not available, and it's a fresh build directory as well, it won't have anywhere to get the previously built files from.

Reply:
No, because the test adds [the mirrors here](https://github.com/mendersoftware/mender-image-tests/blob/698cb01171f7ef1faaabbe8311f21a06cf1a0432/tests/utils/fixtures/fixtures.py#L470). I wrote "cache write" because it actually reads from the cache.

(Also, this alone is not enough, it still runs intro trouble building for example `mender` recipes that depens on other things like `mender-snapshot` (see [CI log here](https://gitlab.com/Northern.tech/Mender/mender-qa/-/jobs/8286531361)), and that is why I also kept the soft clean below)

Reply:
Ok, thanks for explaining! Then it looks good, I think that deserves a comment! 🙂